### ISSUE TITLE
Help: Can't get build to work with different App ID

### ISSUE THREAD

**Issue by blindmonkey** [fastlane/fastlane#11365]
### New Issue Checklist

- [ ] Updated fastlane to the latest version (I'm avoiding doing this because things work with a different App ID and this is likely an error on my part)
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description
I am trying to make our app build with a different App ID. As I understand, this is necessary in order to be able to have a "Beta" build that one can install alongside the production build. We are not using `match` (but will probably look into it at some point in the future), but are using `sigh` instead. We're not explicitly using `cert`, since the proper certificate appears to be in the keychain.

We have had the app building with its normal bundle ID, `com.company.product`, for a while now. But in trying to get the new bundle ID set up, I keep getting the error:
```
▸ === BUILD TARGET Product OF PROJECT Product WITH CONFIGURATION ProductionBeta ===
▸ Check dependencies
▸ Code Signing Error: No profiles for 'com.company.product.production.beta' were found:  Xcode couldn't find any iOS App Development provisioning profiles matching 'com.company.product.production.beta'. Automatic signing is disabled and unable to generate a profile. To enable automatic signing, pass -allowProvisioningUpdates to xcodebuild.
▸ Code Signing Error: Code signing is required for product type 'Application' in SDK 'iOS 11.0'
```
I've tried passing `-allowProvisioningUpdates` in the `xcargs` parameter to `gym`, but that didn't help, and just told me that it wasn't able to log in. `sigh` appears to succeed and get the provisioning profile without issues, so it appears to either be that the profile isn't valid for the app somehow, or something on that end isn't configured properly.

We have a build configurable variable, `BUNDLE_ID`, so to change our app bundle ID, I duplicated our `Release` configuration to a new build configuration called `ProductionBeta`, where `BUNDLE_ID` is set to `com.company.product.production.beta` instead of `com.company.product`. I added a new App ID in Apple's Developer portal which matches that new bundle ID, and configured the App Group to be `group.com.company.product`, since that's what's set for our app. Nothing else should have changed, and I've verified that our `Release` build still works. The `ProductionBeta` configuration appears to build in Xcode as well.

Since we also use a number of `cocoapods`, after creating my new configuration I ran `pod install`, which made some changes to the project.

##### Complete output when running fastlane, including the stack trace and command used

```
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url git@github.lab.mtl:Company/Product.git # timeout=10
Fetching upstream changes from git@github.lab.mtl:Company/Product.git
 > git --version # timeout=10
 > git fetch --tags --progress git@github.lab.mtl:Company/Product.git +refs/heads/*:refs/remotes/origin/*
 > git rev-parse origin/crashlytics-beta^{commit} # timeout=10
Checking out Revision c5a93739d50162aa42d0c93d208bf35fed9137b3 (origin/crashlytics-beta)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f c5a93739d50162aa42d0c93d208bf35fed9137b3
 > git rev-list c5a93739d50162aa42d0c93d208bf35fed9137b3 # timeout=10
 > git remote # timeout=10
 > git submodule init # timeout=10
 > git submodule sync # timeout=10
 > git config --get remote.origin.url # timeout=10
 > git submodule init # timeout=10
 > git config -f .gitmodules --get-regexp ^submodule\.(.*)\.url # timeout=10
 > git config --get submodule.JSONCodable.url # timeout=10
 > git config -f .gitmodules --get submodule.JSONCodable.path # timeout=10
 > git submodule update --init --recursive JSONCodable
[Product Distribution] $ /bin/sh -xe /var/folders/p9/7n11r4ks5xd4hlzs0nzfz5v40000gp/T/hudson937758878184879419.sh
+ export LANG=en_US.UTF-8
+ LANG=en_US.UTF-8
+ export LANGUAGE=en_US.UTF-8
+ LANGUAGE=en_US.UTF-8
+ export LC_ALL=en_US.UTF-8
+ LC_ALL=en_US.UTF-8
+ security unlock-keychain -p $PASSWORD /Users/jenkins/Library/Keychains/login.keychain
+ source /Users/jenkins/.bash_profile
++ PATH=/usr/bin:/bin:/usr/sbin:/sbin:/Users/jenkins/.rbenv/shims:/usr/local/bin/:/usr/local/bin/:/Users/jenkins/.rbenv/shims
++ export ITUNES_USER=email@company.com
++ ITUNES_USER=email@company.com
+++ rbenv init -
++ eval 'export PATH="/Users/jenkins/.rbenv/shims:${PATH}"
source "/usr/local/Cellar/rbenv/0.4.0/libexec/../completions/rbenv.bash"
rbenv rehash 2>/dev/null
rbenv() {
  typeset command
  command="$1"
  if [ "$#" -gt 0 ]; then
    shift
  fi

  case "$command" in
  rehash|shell)
    eval `rbenv "sh-$command" "$@"`;;
  *)
    command rbenv "$command" "$@";;
  esac
}'
+++ export PATH=/Users/jenkins/.rbenv/shims:/usr/bin:/bin:/usr/sbin:/sbin:/Users/jenkins/.rbenv/shims:/usr/local/bin/:/usr/local/bin/:/Users/jenkins/.rbenv/shims
+++ PATH=/Users/jenkins/.rbenv/shims:/usr/bin:/bin:/usr/sbin:/sbin:/Users/jenkins/.rbenv/shims:/usr/local/bin/:/usr/local/bin/:/Users/jenkins/.rbenv/shims
+++ source /usr/local/Cellar/rbenv/0.4.0/libexec/../completions/rbenv.bash
++++ complete -F _rbenv rbenv
+++ rbenv rehash
+ bundle install
Warning: the running version of Bundler is older than the version that created the lockfile. We suggest you upgrade to the latest version of Bundler by running `gem install bundler`.
Using CFPropertyList 2.3.5
Using i18n 0.8.1
Using minitest 5.10.2
Using thread_safe 0.3.6
Using public_suffix 2.0.5
Using babosa 1.0.2
Using claide 1.0.2
Using fuzzy_match 2.0.4
Using nap 1.1.0
Using cocoapods-deintegrate 1.0.1
Using cocoapods-downloader 1.1.3
Using cocoapods-search 1.0.0
Using cocoapods-stats 1.0.0
Using netrc 0.7.8
Using cocoapods-try 1.1.0
Using colored2 3.1.2
Using escape 0.0.4
Using fourflusher 2.0.1
Using gh_inspector 1.0.3
Using molinillo 0.5.7
Using ruby-macho 1.1.0
Using nanaimo 0.2.3
Using colored 1.2
Using highline 1.7.8
Using declarative 0.0.10
Using declarative-option 0.1.0
Using unf_ext 0.0.7.4
Using dotenv 2.2.1
Using excon 0.59.0
Using multipart-post 2.0.0
Using fastimage 2.1.0
Using bundler 1.12.5
Using jwt 1.5.6
Using little-plugger 1.1.4
Using multi_json 1.12.2
Using memoist 0.16.0
Using os 0.9.6
Using httpclient 2.8.3
Using mime-types-data 3.2016.0521
Using uber 0.1.0
Using retriable 3.1.1
Using json 2.1.0
Using mini_magick 4.5.1
Using multi_xml 0.6.0
Using plist 3.3.0
Using rubyzip 1.2.1
Using security 0.1.3
Using slack-notifier 1.5.1
Using terminal-notifier 1.8.0
Using unicode-display_width 1.3.0
Using tty-screen 0.5.0
Using word_wrap 1.0.0
Using rouge 2.0.7
Using tzinfo 1.2.3
Using addressable 2.5.2
Using cocoapods-plugins 1.0.0
Using cocoapods-trunk 1.2.0
Using xcodeproj 1.5.2
Using commander-fastlane 4.4.5
Using unf 0.1.4
Using faraday 0.13.1
Using logging 2.2.2
Using mime-types 3.1
Using representable 3.0.4
Using terminal-table 1.8.0
Using xcpretty 0.2.8
Using activesupport 4.2.8
Using domain_name 0.5.20170404
Using faraday_middleware 0.12.2
Using signet 0.7.3
Using xcpretty-travis-formatter 0.0.4
Using cocoapods-core 1.2.1
Using http-cookie 1.0.3
Using googleauth 0.5.3
Using cocoapods 1.2.1
Using faraday-cookie_jar 0.0.6
Using google-api-client 0.13.6
Using fastlane 2.60.1
Bundle complete! 2 Gemfile dependencies, 78 gems now installed.
Use `bundle show [gemname]` to see where a bundled gem is installed.
+ envchain product env bundle exec fastlane crashlytics_ad_hoc group:urbanairship-testers --capture_output
### Captured Output

Command Used: `crashlytics_ad_hoc group:urbanairship-testers`
<details><summary>Output/Log</summary>

INFO [2017-12-28 09:38:11.80]: Successfully loaded custom action '/Users/jenkins/workspace/Product Distribution/fastlane/actions/cupertino_build_version.rb'.
DEBUG [2017-12-28 09:38:11.80]: Checking if there are any plugins that should be loaded...
DEBUG [2017-12-28 09:38:11.90]: Using group: urbanairship-testers
INFO [2017-12-28 09:38:11.91]: -------------------------------------------------
INFO [2017-12-28 09:38:11.91]: --- Step: Verifying required fastlane version ---
INFO [2017-12-28 09:38:11.91]: -------------------------------------------------
INFO [2017-12-28 09:38:11.91]: Your fastlane version 2.60.1 matches the minimum requirement of 1.105.1  ✅
INFO [2017-12-28 09:38:11.91]: ------------------------------
INFO [2017-12-28 09:38:11.91]: --- Step: default_platform ---
INFO [2017-12-28 09:38:11.91]: ------------------------------
INFO [2017-12-28 09:38:11.91]: Driving the lane 'ios crashlytics_ad_hoc' 🚀
INFO [2017-12-28 09:38:11.91]: --------------------------------
INFO [2017-12-28 09:38:11.91]: --- Step: clear_derived_data ---
INFO [2017-12-28 09:38:11.91]: --------------------------------
INFO [2017-12-28 09:38:11.91]: Derived Data path located at: /Users/jenkins/Library/Developer/Xcode/DerivedData
INFO [2017-12-28 09:38:12.85]: Successfully cleared Derived Data ♻️
INFO [2017-12-28 09:38:12.85]: -------------------
INFO [2017-12-28 09:38:12.85]: --- Step: is_ci ---
INFO [2017-12-28 09:38:12.85]: -------------------
INFO [2017-12-28 09:38:12.86]: -------------------------------------
INFO [2017-12-28 09:38:12.86]: --- Step: ensure_git_status_clean ---
INFO [2017-12-28 09:38:12.86]: -------------------------------------
INFO [2017-12-28 09:38:12.86]: $ git status --porcelain
INFO [2017-12-28 09:38:12.97]: Git status is clean, all good! 💪
INFO [2017-12-28 09:38:12.97]: ---------------------------------------------
INFO [2017-12-28 09:38:12.97]: --- Step: Switch to ios ad_hoc_build lane ---
INFO [2017-12-28 09:38:12.97]: ---------------------------------------------
INFO [2017-12-28 09:38:12.97]: Cruising over to lane 'ios ad_hoc_build' 🚖
INFO [2017-12-28 09:38:12.98]: -----------------------------------------------------
INFO [2017-12-28 09:38:12.98]: Step: Switch to ios update_target_version lane
INFO [2017-12-28 09:38:12.98]: -----------------------------------------------------
INFO [2017-12-28 09:38:12.98]: Cruising over to lane 'ios update_target_version' 🚖
INFO [2017-12-28 09:38:12.98]: ---------------------------------------------
INFO [2017-12-28 09:38:12.98]: --- Step: ../bin/update-target-version.sh ---
INFO [2017-12-28 09:38:12.98]: ---------------------------------------------
INFO [2017-12-28 09:38:12.98]: $ ../bin/update-target-version.sh
INFO [2017-12-28 09:38:13.09]: Cruising back to lane 'ios ad_hoc_build' 🚘
INFO [2017-12-28 09:38:13.09]: ------------------
INFO [2017-12-28 09:38:13.09]: --- Step: sigh ---
INFO [2017-12-28 09:38:13.09]: ------------------
Successfully loaded Appfile at path '/Users/jenkins/workspace/Product Distribution/fastlane/Appfile'
- app_identifier: 'com.company.product'
- team_name: 'Company Inc.'
- team_id: 'TEAMID'
-------
Successfully loaded Appfile at path '/Users/jenkins/workspace/Product Distribution/fastlane/Appfile'
- app_identifier: 'com.company.product'
- team_name: 'Company Inc.'
- team_id: 'TEAMID'
-------
Successfully loaded Appfile at path '/Users/jenkins/workspace/Product Distribution/fastlane/Appfile'
- app_identifier: 'com.company.product'
- team_name: 'Company Inc.'
- team_id: 'TEAMID'
-------
Successfully loaded Appfile at path '/Users/jenkins/workspace/Product Distribution/fastlane/Appfile'
- app_identifier: 'com.company.product'
- team_name: 'Company Inc.'
- team_id: 'TEAMID'
-------
Successfully loaded Appfile at path '/Users/jenkins/workspace/Product Distribution/fastlane/Appfile'
- app_identifier: 'com.company.product'
- team_name: 'Company Inc.'
- team_id: 'TEAMID'
-------
Successfully loaded Appfile at path '/Users/jenkins/workspace/Product Distribution/fastlane/Appfile'
- app_identifier: 'com.company.product'
- team_name: 'Company Inc.'
- team_id: 'TEAMID'
-------
Successfully loaded Appfile at path '/Users/jenkins/workspace/Product Distribution/fastlane/Appfile'
- app_identifier: 'com.company.product'
- team_name: 'Company Inc.'
- team_id: 'TEAMID'
-------
Successfully loaded Appfile at path '/Users/jenkins/workspace/Product Distribution/fastlane/Appfile'
- app_identifier: 'com.company.product'
- team_name: 'Company Inc.'
- team_id: 'TEAMID'
-------
Successfully loaded Appfile at path '/Users/jenkins/workspace/Product Distribution/fastlane/Appfile'
- app_identifier: 'com.company.product'
- team_name: 'Company Inc.'
- team_id: 'TEAMID'
-------
Successfully loaded Appfile at path '/Users/jenkins/workspace/Product Distribution/fastlane/Appfile'
- app_identifier: 'com.company.product'
- team_name: 'Company Inc.'
- team_id: 'TEAMID'
-------
DEBUG [2017-12-28 09:38:13.10]: Taking value for 'username' from environment variable 'SIGH_USERNAME'
DEBUG [2017-12-28 09:38:13.10]: Taking value for 'output_path' from environment variable 'SIGH_OUTPUT_PATH'

+-------------------------------------+------------------------------------+
|                         Summary for sigh 2.60.1                          |
+-------------------------------------+------------------------------------+
| adhoc                               | true                               |
| app_identifier                      | com.company.product.production.beta |
| development                         | false                              |
| skip_install                        | false                              |
| force                               | false                              |
| username                            | email@company.com                 |
| team_id                             | TEAMID                         |
| team_name                           | Company Inc.                        |
| ignore_profiles_with_different_name | false                              |
| skip_fetch_profiles                 | false                              |
| skip_certificate_verification       | false                              |
| platform                            | ios                                |
+-------------------------------------+------------------------------------+

INFO [2017-12-28 09:38:13.10]: Starting login with user 'email@company.com'
INFO [2017-12-28 09:38:14.87]: Successfully logged in
INFO [2017-12-28 09:38:14.87]: Fetching profiles...
INFO [2017-12-28 09:38:15.33]: Provisioning Profile 'com.company.product.production.beta AdHoc 1514407347' is not valid, skipping this one...
INFO [2017-12-28 09:38:15.33]: Provisioning Profile 'com.company.product.production.beta AdHoc 1514400931' is not valid, skipping this one...
INFO [2017-12-28 09:38:15.33]: Provisioning Profile 'com.company.product.production.beta AdHoc 1514403158' is not valid, skipping this one...
INFO [2017-12-28 09:38:15.33]: Provisioning Profile 'com.company.product.production.beta AdHoc 1514399273' is not valid, skipping this one...
INFO [2017-12-28 09:38:15.33]: Provisioning Profile 'com.company.product.production.beta AdHoc' is not valid, skipping this one...
INFO [2017-12-28 09:38:15.71]: Verifying certificates...
INFO [2017-12-28 09:38:16.15]: $ security list-keychains -d user
INFO [2017-12-28 09:38:16.16]: ▸     "/Users/jenkins/Library/Keychains/login.keychain-db"
INFO [2017-12-28 09:38:16.16]: $ security find-certificate -c 'Apple Worldwide Developer Relations Certification Authority' /Users/jenkins/Library/Keychains/login.keychain-db
INFO [2017-12-28 09:38:16.18]: ▸ keychain: "/Users/jenkins/Library/Keychains/login.keychain-db"
INFO [2017-12-28 09:38:16.18]: ▸ version: 512
INFO [2017-12-28 09:38:16.18]: ▸ class: 0x80001000 
INFO [2017-12-28 09:38:16.18]: ▸ attributes:
INFO [2017-12-28 09:38:16.18]: ▸     "alis"<blob>="Apple Worldwide Developer Relations Certification Authority"
INFO [2017-12-28 09:38:16.18]: ▸     "cenc"<uint32>=0x00000003 
INFO [2017-12-28 09:38:16.18]: ▸     "ctyp"<uint32>=0x00000001 
INFO [2017-12-28 09:38:16.18]: ▸     "hpky"<blob>=0x88271709A9B618608BECEBBAF64759C55254A3B7  "\210'\027\011\251\266\030`\213\354\353\272\366GY\305RT\243\267"
INFO [2017-12-28 09:38:16.18]: ▸     "issu"<blob>=0x3062310B300906035504061302555331133011060355040A130A4150504C4520494E432E31263024060355040B131D4150504C452043455254494649434154494F4E20415554484F52495459311630140603550403130D4150504C4520524F4F54204341  "0b1\0130\011\006\003U\004\006\023\002US1\0230\021\006\003U\004\012\023\012APPLE INC.1&0$\006\003U\004\013\023\035APPLE CERTIFICATION AUTHORITY1\0260\024\006\003U\004\003\023\015APPLE ROOT CA"
INFO [2017-12-28 09:38:16.18]: ▸     "labl"<blob>="Apple Worldwide Developer Relations Certification Authority"
INFO [2017-12-28 09:38:16.18]: ▸     "skid"<blob>=0x88271709A9B618608BECEBBAF64759C55254A3B7  "\210'\027\011\251\266\030`\213\354\353\272\366GY\305RT\243\267"
INFO [2017-12-28 09:38:16.18]: ▸     "snbr"<blob>=0x19 
INFO [2017-12-28 09:38:16.18]: ▸     "subj"<blob>=0x308196310B300906035504061302555331133011060355040A0C0A4170706C6520496E632E312C302A060355040B0C234170706C6520576F726C647769646520446576656C6F7065722052656C6174696F6E733144304206035504030C3B4170706C6520576F726C647769646520446576656C6F7065722052656C6174696F6E732043657274696669636174696F6E20417574686F72697479  "0\201\2261\0130\011\006\003U\004\006\023\002US1\0230\021\006\003U\004\012\014\012Apple Inc.1,0*\006\003U\004\013\014#Apple Worldwide Developer Relations1D0B\006\003U\004\003\014;Apple Worldwide Developer Relations Certification Authority"
INFO [2017-12-28 09:38:16.70]: Found 1 matching profile(s)
WARN [2017-12-28 09:38:16.70]: Downloading provisioning profile...
INFO [2017-12-28 09:38:17.84]: Successfully downloaded provisioning profile...
INFO [2017-12-28 09:38:17.88]: Installing provisioning profile...
/Users/jenkins/Library/MobileDevice/Provisioning Profiles/AdHoc_com.company.product.production.beta.mobileprovision
INFO [2017-12-28 09:38:17.93]: Setting Provisioning Profile type to 'ad-hoc'
INFO [2017-12-28 09:38:17.93]: ------------------
INFO [2017-12-28 09:38:17.93]: --- Step: sigh ---
INFO [2017-12-28 09:38:17.93]: ------------------
Successfully loaded Appfile at path '/Users/jenkins/workspace/Product Distribution/fastlane/Appfile'
- app_identifier: 'com.company.product'
- team_name: 'Company Inc.'
- team_id: 'TEAMID'
-------
Successfully loaded Appfile at path '/Users/jenkins/workspace/Product Distribution/fastlane/Appfile'
- app_identifier: 'com.company.product'
- team_name: 'Company Inc.'
- team_id: 'TEAMID'
-------
Successfully loaded Appfile at path '/Users/jenkins/workspace/Product Distribution/fastlane/Appfile'
- app_identifier: 'com.company.product'
- team_name: 'Company Inc.'
- team_id: 'TEAMID'
-------
Successfully loaded Appfile at path '/Users/jenkins/workspace/Product Distribution/fastlane/Appfile'
- app_identifier: 'com.company.product'
- team_name: 'Company Inc.'
- team_id: 'TEAMID'
-------
Successfully loaded Appfile at path '/Users/jenkins/workspace/Product Distribution/fastlane/Appfile'
- app_identifier: 'com.company.product'
- team_name: 'Company Inc.'
- team_id: 'TEAMID'
-------
Successfully loaded Appfile at path '/Users/jenkins/workspace/Product Distribution/fastlane/Appfile'
- app_identifier: 'com.company.product'
- team_name: 'Company Inc.'
- team_id: 'TEAMID'
-------
Successfully loaded Appfile at path '/Users/jenkins/workspace/Product Distribution/fastlane/Appfile'
- app_identifier: 'com.company.product'
- team_name: 'Company Inc.'
- team_id: 'TEAMID'
-------
Successfully loaded Appfile at path '/Users/jenkins/workspace/Product Distribution/fastlane/Appfile'
- app_identifier: 'com.company.product'
- team_name: 'Company Inc.'
- team_id: 'TEAMID'
-------
Successfully loaded Appfile at path '/Users/jenkins/workspace/Product Distribution/fastlane/Appfile'
- app_identifier: 'com.company.product'
- team_name: 'Company Inc.'
- team_id: 'TEAMID'
-------
Successfully loaded Appfile at path '/Users/jenkins/workspace/Product Distribution/fastlane/Appfile'
- app_identifier: 'com.company.product'
- team_name: 'Company Inc.'
- team_id: 'TEAMID'
-------
DEBUG [2017-12-28 09:38:17.93]: Taking value for 'username' from environment variable 'SIGH_USERNAME'
DEBUG [2017-12-28 09:38:17.93]: Taking value for 'output_path' from environment variable 'SIGH_OUTPUT_PATH'

+-------------------------------------+---------------------------------------------+
|                              Summary for sigh 2.60.1                              |
+-------------------------------------+---------------------------------------------+
| adhoc                               | true                                        |
| app_identifier                      | com.company.product.production.beta.Stickers |
| development                         | false                                       |
| skip_install                        | false                                       |
| force                               | false                                       |
| username                            | email@company.com                          |
| team_id                             | TEAMID                                  |
| team_name                           | Company Inc.                                 |
| ignore_profiles_with_different_name | false                                       |
| skip_fetch_profiles                 | false                                       |
| skip_certificate_verification       | false                                       |
| platform                            | ios                                         |
+-------------------------------------+---------------------------------------------+

INFO [2017-12-28 09:38:17.94]: Starting login with user 'email@company.com'
INFO [2017-12-28 09:38:19.33]: Successfully logged in
INFO [2017-12-28 09:38:19.33]: Fetching profiles...
INFO [2017-12-28 09:38:19.76]: Provisioning Profile 'com.company.product.production.beta.Stickers AdHoc' is not valid, skipping this one...
INFO [2017-12-28 09:38:20.16]: Verifying certificates...
INFO [2017-12-28 09:38:20.54]: $ security list-keychains -d user
INFO [2017-12-28 09:38:20.55]: ▸     "/Users/jenkins/Library/Keychains/login.keychain-db"
INFO [2017-12-28 09:38:20.55]: $ security find-certificate -c 'Apple Worldwide Developer Relations Certification Authority' /Users/jenkins/Library/Keychains/login.keychain-db
INFO [2017-12-28 09:38:20.57]: ▸ keychain: "/Users/jenkins/Library/Keychains/login.keychain-db"
INFO [2017-12-28 09:38:20.57]: ▸ version: 512
INFO [2017-12-28 09:38:20.57]: ▸ class: 0x80001000 
INFO [2017-12-28 09:38:20.57]: ▸ attributes:
INFO [2017-12-28 09:38:20.57]: ▸     "alis"<blob>="Apple Worldwide Developer Relations Certification Authority"
INFO [2017-12-28 09:38:20.57]: ▸     "cenc"<uint32>=0x00000003 
INFO [2017-12-28 09:38:20.57]: ▸     "ctyp"<uint32>=0x00000001 
INFO [2017-12-28 09:38:20.57]: ▸     "hpky"<blob>=0x88271709A9B618608BECEBBAF64759C55254A3B7  "\210'\027\011\251\266\030`\213\354\353\272\366GY\305RT\243\267"
INFO [2017-12-28 09:38:20.57]: ▸     "issu"<blob>=0x3062310B300906035504061302555331133011060355040A130A4150504C4520494E432E31263024060355040B131D4150504C452043455254494649434154494F4E20415554484F52495459311630140603550403130D4150504C4520524F4F54204341  "0b1\0130\011\006\003U\004\006\023\002US1\0230\021\006\003U\004\012\023\012APPLE INC.1&0$\006\003U\004\013\023\035APPLE CERTIFICATION AUTHORITY1\0260\024\006\003U\004\003\023\015APPLE ROOT CA"
INFO [2017-12-28 09:38:20.57]: ▸     "labl"<blob>="Apple Worldwide Developer Relations Certification Authority"
INFO [2017-12-28 09:38:20.57]: ▸     "skid"<blob>=0x88271709A9B618608BECEBBAF64759C55254A3B7  "\210'\027\011\251\266\030`\213\354\353\272\366GY\305RT\243\267"
INFO [2017-12-28 09:38:20.57]: ▸     "snbr"<blob>=0x19 
INFO [2017-12-28 09:38:20.57]: ▸     "subj"<blob>=0x308196310B300906035504061302555331133011060355040A0C0A4170706C6520496E632E312C302A060355040B0C234170706C6520576F726C647769646520446576656C6F7065722052656C6174696F6E733144304206035504030C3B4170706C6520576F726C647769646520446576656C6F7065722052656C6174696F6E732043657274696669636174696F6E20417574686F72697479  "0\201\2261\0130\011\006\003U\004\006\023\002US1\0230\021\006\003U\004\012\014\012Apple Inc.1,0*\006\003U\004\013\014#Apple Worldwide Developer Relations1D0B\006\003U\004\003\014;Apple Worldwide Developer Relations Certification Authority"
INFO [2017-12-28 09:38:21.09]: Found 1 matching profile(s)
WARN [2017-12-28 09:38:21.09]: Downloading provisioning profile...
INFO [2017-12-28 09:38:22.23]: Successfully downloaded provisioning profile...
INFO [2017-12-28 09:38:22.27]: Installing provisioning profile...
/Users/jenkins/Library/MobileDevice/Provisioning Profiles/AdHoc_com.company.product.production.beta.Stickers.mobileprovision
INFO [2017-12-28 09:38:22.31]: Setting Provisioning Profile type to 'ad-hoc'
INFO [2017-12-28 09:38:22.31]: -----------------
INFO [2017-12-28 09:38:22.31]: --- Step: gym ---
INFO [2017-12-28 09:38:22.31]: -----------------
INFO [2017-12-28 09:38:22.44]: Successfully loaded '/Users/jenkins/workspace/Product Distribution/Gymfile' 📄

+------------------+---------------+
| Detected Values from './Gymfile' |
+------------------+---------------+
| scheme           | Product       |
| output_directory | ./build       |
| clean            | true          |
| output_name      | Product       |
+------------------+---------------+

INFO [2017-12-28 09:38:22.44]: Successfully loaded '/Users/jenkins/workspace/Product Distribution/Gymfile' 📄
WARN [2017-12-28 09:38:22.44]: No values defined in './Gymfile'
INFO [2017-12-28 09:38:22.44]: $ xcodebuild -list -workspace ./Product.xcworkspace -configuration ProductionBeta
INFO [2017-12-28 09:38:23.78]: $ xcodebuild -showBuildSettings -workspace ./Product.xcworkspace -scheme Product -configuration ProductionBeta
DEBUG [2017-12-28 09:38:27.12]: Parsing project file '/Users/jenkins/workspace/Product Distribution/Product.xcodeproj' to find selected provisioning profiles
DEBUG [2017-12-28 09:38:27.12]: Finding provision profiles for 'ProductionBeta'
DEBUG [2017-12-28 09:38:27.37]: Merging provisioning profile mappings
DEBUG [2017-12-28 09:38:27.37]: -------------------------------------
DEBUG [2017-12-28 09:38:27.37]: Primary provisioning profile mapping:
DEBUG [2017-12-28 09:38:27.37]: {"com.company.product.production.beta"=>"com.company.product.production.beta AdHoc 1514470392", "com.company.product.production.beta.Stickers"=>"com.company.product.production.beta.Stickers AdHoc 1514392696"}
DEBUG [2017-12-28 09:38:27.37]: Secondary provisioning profile mapping:
DEBUG [2017-12-28 09:38:27.37]: {}
DEBUG [2017-12-28 09:38:27.37]: Resulting in the following mapping:
DEBUG [2017-12-28 09:38:27.37]: {:"com.company.product.production.beta"=>"com.company.product.production.beta AdHoc 1514470392", :"com.company.product.production.beta.Stickers"=>"com.company.product.production.beta.Stickers AdHoc 1514392696"}
INFO [2017-12-28 09:38:27.37]: Detected provisioning profile mapping: {:"com.company.product.production.beta"=>"com.company.product.production.beta AdHoc 1514470392", :"com.company.product.production.beta.Stickers"=>"com.company.product.production.beta.Stickers AdHoc 1514392696"}

+---------------------------------------------------------------------------------+--------------------------------------------------------------+
|                                                             Summary for gym 2.60.1                                                             |
+---------------------------------------------------------------------------------+--------------------------------------------------------------+
| export_xcargs                                                                   | -allowProvisioningUpdates                                    |
| xcconfig                                                                        | Beta.xcconfig                                                |
| configuration                                                                   | ProductionBeta                                               |
| export_method                                                                   | ad-hoc                                                       |
| export_options.signingStyle                                                     | manual                                                       |
| export_options.provisioningProfiles.com.company.product.production.beta          | com.company.product.production.beta AdHoc 1514470392          |
| export_options.provisioningProfiles.com.company.product.production.beta.Stickers | com.company.product.production.beta.Stickers AdHoc 1514392696 |
| scheme                                                                          | Product                                                      |
| output_directory                                                                | ./build                                                      |
| clean                                                                           | true                                                         |
| output_name                                                                     | Product                                                      |
| workspace                                                                       | ./Product.xcworkspace                                        |
| destination                                                                     | generic/platform=iOS                                         |
| build_path                                                                      | /Users/jenkins/Library/Developer/Xcode/Archives/2017-12-28   |
| silent                                                                          | false                                                        |
| skip_package_ipa                                                                | false                                                        |
| buildlog_path                                                                   | ~/Library/Logs/gym                                           |
| xcode_path                                                                      | /Applications/Xcode.app                                      |
+---------------------------------------------------------------------------------+--------------------------------------------------------------+

+-----------------------------------------------------------------------------------------------------------------+----------------------+
|                                                        Generated Build Command                                                         |
+-----------------------------------------------------------------------------------------------------------------+----------------------+
| Option                                                                                                          | Value                |
+-----------------------------------------------------------------------------------------------------------------+----------------------+
| set -o pipefail &&                                                                                              |                      |
| xcodebuild                                                                                                      |                      |
| -workspace ./Product.xcworkspace                                                                                |                      |
| -scheme Product                                                                                                 |                      |
| -configuration ProductionBeta                                                                                   |                      |
| -destination                                                                                                    | generic/platform=iOS |
| -xcconfig                                                                                                       | Beta.xcconfig        |
| -archivePath /Users/jenkins/Library/Developer/Xcode/Archives/2017-12-28/Product\ 2017-12-28\ 09.38.27.xcarchive |                      |
| clean                                                                                                           |                      |
| archive                                                                                                         |                      |
| | tee /Users/jenkins/Library/Logs/gym/Product-Product.log                                                       |                      |
| | xcpretty                                                                                                      |                      |
+-----------------------------------------------------------------------------------------------------------------+----------------------+
INFO [2017-12-28 09:38:27.39]: $ set -o pipefail && xcodebuild -workspace ./Product.xcworkspace -scheme Product -configuration ProductionBeta -destination 'generic/platform=iOS' -xcconfig 'Beta.xcconfig' -archivePath /Users/jenkins/Library/Developer/Xcode/Archives/2017-12-28/Product\ 2017-12-28\ 09.38.27.xcarchive clean archive | tee /Users/jenkins/Library/Logs/gym/Product-Product.log | xcpretty
INFO [2017-12-28 09:38:29.43]: ▸ Cleaning Product/Stickers [ProductionBeta]
INFO [2017-12-28 09:38:29.46]: ▸ Check Dependencies
INFO [2017-12-28 09:38:29.57]: ▸ Cleaning Pods/AFNetworking [ProductionBeta]
INFO [2017-12-28 09:38:29.57]: ▸ Check Dependencies
INFO [2017-12-28 09:38:29.57]: ▸ Cleaning Pods/Bolts [ProductionBeta]
INFO [2017-12-28 09:38:29.57]: ▸ Check Dependencies
INFO [2017-12-28 09:38:29.57]: ▸ Cleaning Pods/Result [ProductionBeta]
INFO [2017-12-28 09:38:29.57]: ▸ Check Dependencies
INFO [2017-12-28 09:38:29.57]: ▸ Cleaning Pods/Collector [ProductionBeta]
INFO [2017-12-28 09:38:29.57]: ▸ Check Dependencies
INFO [2017-12-28 09:38:29.57]: ▸ Cleaning Pods/Dwifft [ProductionBeta]
INFO [2017-12-28 09:38:29.57]: ▸ Check Dependencies
INFO [2017-12-28 09:38:29.57]: ▸ Cleaning Pods/FBSDKCoreKit [ProductionBeta]
INFO [2017-12-28 09:38:29.57]: ▸ Check Dependencies
INFO [2017-12-28 09:38:30.03]: ▸ Cleaning Pods/BrightFutures [ProductionBeta]
INFO [2017-12-28 09:38:30.03]: ▸ Check Dependencies
INFO [2017-12-28 09:38:30.03]: ▸ Cleaning Pods/FBSDKMessengerShareKit [ProductionBeta]
INFO [2017-12-28 09:38:30.03]: ▸ Check Dependencies
INFO [2017-12-28 09:38:30.04]: ▸ Cleaning Pods/FlexiblePageControl [ProductionBeta]
INFO [2017-12-28 09:38:30.04]: ▸ Check Dependencies
INFO [2017-12-28 09:38:30.04]: ▸ Cleaning Pods/GzipSwift [ProductionBeta]
INFO [2017-12-28 09:38:30.04]: ▸ Check Dependencies
INFO [2017-12-28 09:38:30.04]: ▸ Cleaning Pods/ICTutorialOverlay [ProductionBeta]
INFO [2017-12-28 09:38:30.04]: ▸ Check Dependencies
INFO [2017-12-28 09:38:30.04]: ▸ Cleaning Pods/ISO8601DateFormatter [ProductionBeta]
INFO [2017-12-28 09:38:30.04]: ▸ Check Dependencies
INFO [2017-12-28 09:38:30.04]: ▸ Cleaning Pods/JSONCodable [ProductionBeta]
INFO [2017-12-28 09:38:30.04]: ▸ Check Dependencies
INFO [2017-12-28 09:38:30.04]: ▸ Cleaning Pods/Mantle [ProductionBeta]
INFO [2017-12-28 09:38:30.04]: ▸ Check Dependencies
INFO [2017-12-28 09:38:30.04]: ▸ Cleaning Pods/Mixpanel [ProductionBeta]
INFO [2017-12-28 09:38:30.04]: ▸ Check Dependencies
INFO [2017-12-28 09:38:30.22]: ▸ Cleaning Pods/PaymentKit [ProductionBeta]
INFO [2017-12-28 09:38:30.22]: ▸ Check Dependencies
INFO [2017-12-28 09:38:30.22]: ▸ Cleaning Pods/Slash [ProductionBeta]
INFO [2017-12-28 09:38:30.22]: ▸ Check Dependencies
INFO [2017-12-28 09:38:30.22]: ▸ Cleaning Pods/Valet [ProductionBeta]
INFO [2017-12-28 09:38:30.22]: ▸ Check Dependencies
INFO [2017-12-28 09:38:30.22]: ▸ Cleaning Pods/libPhoneNumber-iOS [ProductionBeta]
INFO [2017-12-28 09:38:30.22]: ▸ Check Dependencies
INFO [2017-12-28 09:38:30.22]: ▸ Cleaning Pods/FBSDKShareKit [ProductionBeta]
INFO [2017-12-28 09:38:30.22]: ▸ Check Dependencies
INFO [2017-12-28 09:38:30.22]: ▸ Cleaning Pods/Pods-ProductAndTests-Product [ProductionBeta]
INFO [2017-12-28 09:38:30.22]: ▸ Check Dependencies
INFO [2017-12-28 09:38:30.22]: ▸ Cleaning Product/Product [ProductionBeta]
INFO [2017-12-28 09:38:30.22]: ▸ Check Dependencies
INFO [2017-12-28 09:38:32.29]: ▸ Clean Succeeded
INFO [2017-12-28 09:38:32.64]: ▸ Building Product/Stickers [ProductionBeta]
INFO [2017-12-28 09:38:32.70]: ▸ Check Dependencies
INFO [2017-12-28 09:38:37.44]: ▸ Processing Info.plist
INFO [2017-12-28 09:38:43.31]: ▸ Touching Stickers.appex
INFO [2017-12-28 09:38:43.99]: ▸ Signing /Users/jenkins/Library/Developer/Xcode/DerivedData/Product-fdyzegbogebcurarnimudkvdfmai/Build/Intermediates.noindex/ArchiveIntermediates/Product/IntermediateBuildFilesPath/UninstalledProducts/iphoneos/Stickers.appex
INFO [2017-12-28 09:38:44.27]: ▸ Building Pods/AFNetworking [ProductionBeta]
INFO [2017-12-28 09:38:44.27]: ▸ Check Dependencies
INFO [2017-12-28 09:38:44.27]: ▸ Processing Info.plist
INFO [2017-12-28 09:38:44.27]: ▸ Compiling...
▸ Copying Pods-ProductAndTests-Product-umbrella.h
▸ Touching Pods_ProductAndTests_Product.framework
▸ Building Product/Product [ProductionBeta]
▸ Check Dependencies
** ARCHIVE FAILED **


The following build commands failed:
Check dependencies
(1 failure)
ERROR [2017-12-28 09:39:49.94]: Exit status: 65

+---------------+-------------------------+
|            Build environment            |
+---------------+-------------------------+
| xcode_path    | /Applications/Xcode.app |
| gym_version   | 2.60.1                  |
| export_method | ad-hoc                  |
| sdk           | iPhoneOS11.0.sdk        |
+---------------+-------------------------+

INFO [2017-12-28 09:39:49.97]: ▸ === BUILD TARGET Product OF PROJECT Product WITH CONFIGURATION ProductionBeta ===
INFO [2017-12-28 09:39:49.97]: ▸ Check dependencies
INFO [2017-12-28 09:39:49.97]: ▸ Code Signing Error: No profiles for 'com.company.product.production.beta' were found:  Xcode couldn't find any iOS App Development provisioning profiles matching 'com.company.product.production.beta'. Automatic signing is disabled and unable to generate a profile. To enable automatic signing, pass -allowProvisioningUpdates to xcodebuild.
INFO [2017-12-28 09:39:49.97]: ▸ Code Signing Error: Code signing is required for product type 'Application' in SDK 'iOS 11.0'
INFO [2017-12-28 09:39:49.97]: 
ERROR [2017-12-28 09:39:49.97]: ⬆️  Check out the few lines of raw `xcodebuild` output above for potential hints on how to solve this error
WARN [2017-12-28 09:39:49.97]: 📋  For the complete and more detailed error log, check the full log at:
WARN [2017-12-28 09:39:49.97]: 📋  /Users/jenkins/Library/Logs/gym/Product-Product.log
INFO [2017-12-28 09:39:49.97]: 
ERROR [2017-12-28 09:39:49.97]: Looks like fastlane ran into a build/archive error with your project
ERROR [2017-12-28 09:39:49.97]: It's hard to tell what's causing the error, so we wrote some guides on how
ERROR [2017-12-28 09:39:49.97]: to troubleshoot build and signing issues: https://docs.fastlane.tools/codesigning/getting-started/
ERROR [2017-12-28 09:39:49.97]: Before submitting an issue on GitHub, please follow the guide above and make
ERROR [2017-12-28 09:39:49.97]: sure your project is set up correctly.
ERROR [2017-12-28 09:39:49.97]: fastlane uses `xcodebuild` commands to generate your binary, you can see the
ERROR [2017-12-28 09:39:49.97]: the full commands printed out in yellow in the above log.
ERROR [2017-12-28 09:39:49.97]: Make sure to inspect the output above, as usually you'll find more error information there
INFO [2017-12-28 09:39:49.97]: 
WARN [2017-12-28 09:39:49.97]: Lane Context:
INFO [2017-12-28 09:39:49.97]: {:DEFAULT_PLATFORM=>:ios, :PLATFORM_NAME=>:ios, :LANE_NAME=>"ios crashlytics_ad_hoc", :GIT_REPO_WAS_CLEAN_ON_START=>true, :SIGH_PROFILE_PATH=>"/Users/jenkins/Library/MobileDevice/Provisioning Profiles/AdHoc_com.company.product.production.beta.Stickers.mobileprovision", :SIGH_PROFILE_PATHS=>["/Users/jenkins/Library/MobileDevice/Provisioning Profiles/AdHoc_com.company.product.production.beta.mobileprovision", "/Users/jenkins/Library/MobileDevice/Provisioning Profiles/AdHoc_com.company.product.production.beta.Stickers.mobileprovision"], :SIGH_UDID=>"e30cf142-8b61-4c27-b900-e0fa0296965b", :SIGH_UUID=>"e30cf142-8b61-4c27-b900-e0fa0296965b", :SIGH_PROFILE_TYPE=>"ad-hoc"}
ERROR [2017-12-28 09:39:49.97]: Error building the application - see the log above
INFO [2017-12-28 09:39:49.97]: Successfully generated documentation at path '/Users/jenkins/workspace/Product Distribution/fastlane/README.md'

+------+------------------------------------------+-------------+
|                       fastlane summary                        |
+------+------------------------------------------+-------------+
| Step | Action                                   | Time (in s) |
+------+------------------------------------------+-------------+
| 1    | Verifying required fastlane version      | 0           |
| 2    | default_platform                         | 0           |
| 3    | clear_derived_data                       | 0           |
| 4    | is_ci                                    | 0           |
| 5    | ensure_git_status_clean                  | 0           |
| 6    | Switch to ios ad_hoc_build lane          | 0           |
| 7    | Switch to ios update_target_version lane | 0           |
| 8    | ../bin/update-target-version.sh          | 0           |
| 9    | sigh                                     | 4           |
| 10   | sigh                                     | 4           |
| 💥   | gym                                      | 87          |
+------+------------------------------------------+-------------+

ERROR [2017-12-28 09:39:49.98]: fastlane finished with errors

</details>

<details><summary>🚫 fastlane environment 🚫</summary>

### Stack

| Key                         | Value                                       |
| --------------------------- | ------------------------------------------- |
| OS                          | 10.12.6                                     |
| Ruby                        | 2.2.2                                       |
| Bundler?                    | true                                        |
| Git                         | git version 2.13.5 (Apple Git-94)           |
| Installation Source         | ~/.rbenv/versions/2.2.2/bin/fastlane        |
| Host                        | Mac OS X 10.12.6 (16G29)                    |
| Ruby Lib Dir                | ~/.rbenv/versions/2.2.2/lib                 |
| OpenSSL Version             | OpenSSL 1.0.2d 9 Jul 2015                   |
| Is contained                | false                                       |
| Is homebrew                 | false                                       |
| Is installed via Fabric.app | false                                       |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version               | 9.0.1                                       |


### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   | en_US.UTF-8 | ✅ |
| LANGUAGE | en_US.UTF-8 | ✅ |


### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

``ruby
# Customise this file, documentation can be found here:
# https://github.com/KrauseFx/fastlane/tree/master/docs
# All available actions: https://github.com/KrauseFx/fastlane/blob/master/docs/Actions.md
# can also be listed using the `fastlane actions` command

# Change the syntax highlighting to Ruby
# All lines starting with a # are ignored when running `fastlane`

# By default, fastlane will send which actions are used
# No personal data is shared, more information on https://github.com/fastlane/enhancer
# Uncomment the following line to opt out
# opt_out_usage

# If you want to automatically update fastlane if a new version is available:
# update_fastlane

# This is the minimum version number required.
# Update this, if you use features of a newer version
fastlane_version "1.105.1"

default_platform :ios

platform :ios do

  default_username = ENV["ITUNES_USER"] || "#{sh("whoami").strip}@company.com"
  ENV["SIGH_OUTPUT_PATH"] = "~/Library/MobileDevice/Provisioning Profiles"
  ENV["SIGH_USERNAME"] ||= default_username

  before_all do
    clear_derived_data
    if is_ci?
      ensure_git_status_clean
    end
  end

  lane :build do |options|
    target_sighs
    gym(options.merge(:export_xcargs => "-allowProvisioningUpdates", :export_method => "development"))
  end

  lane :ad_hoc_build do |options|
    update_target_version
    target_sighs(:adhoc => true,
                 :app_identifier => 'com.company.product.production.beta')
    gym(options.merge(:export_xcargs => "-allowProvisioningUpdates",
                      :xcconfig => "Beta.xcconfig",
                      :configuration => "ProductionBeta",
                      :export_method => "ad-hoc",
                      :export_options => { :signingStyle => "manual" }))
  end

  lane :device do |options|
    target_sighs(:development => true)
    gym(options)
    install_on_device
  end

  desc "Distribute a crashlytics staging build"
  lane :crashlytics_staging do |options|
    update_target_version
    target_sighs(:adhoc => true)
    gym(:export_xcargs => "-allowProvisioningUpdates",
        :xcconfig => "Distribution.xcconfig",
        :export_method => "ad-hoc",
        :export_options => { :signingStyle => "manual" })
    group = options[:group] || "internal-testers"
    crashlytics(:notes => changelog,
                :groups => [group])
    tag
  end

  desc "Distribute a crashlytics production build"
  lane :crashlytics_ad_hoc do |options|
    ad_hoc_build
    group = options[:group] || "internal-testers"
    crashlytics(:notes => changelog,
               :groups => [group])
    tag
  end

  desc "Deploy a new Beta Version to iTtunesConnect"
  lane :itc_beta do
    update_target_version
    target_sighs
    gym(:export_xcargs => "-allowProvisioningUpdates", :xcconfig => "Beta.xcconfig")
    deliver(force: true)
    tag
  end

  desc "Deploy a new App-Store ready version to iTunesConnect"
  lane :app_store do
    ensure_git_branch(:branch => "stable")
    update_target_version
    target_sighs
    gym(:export_xcargs => "-allowProvisioningUpdates",
        :xcconfig => "AppStore.xcconfig",
        :export_method => "app-store",
        :export_options => { :signingStyle => "manual" })
    deliver(:force => true)
    tag
  end

  desc "Run unit tests for Product"
  lane :test do
    scan(scheme: "Product")
  end

  private_lane :changelog do
    tag = last_git_tag.strip
    "Changes Since #{tag}:\n#{sh "git log --oneline --decorate #{tag}..HEAD"}"
  end

  private_lane :tag do
    t = tag_name(get_version_number, cupertino_build_version, lane_context)
    add_git_tag(:tag => t)
    sh("git push origin #{t}")
  end

  private_lane :update_target_version do 
    sh "../bin/update-target-version.sh"
  end

  def tag_prefix(c)
    dir = c[SharedValues::LANE_NAME].split(" ").last
    "builds/#{dir}"
  end

  def tag_name(version, number, c)
    "#{tag_prefix(c)}/#{version}-#{number}"
  end

  def target_sighs(sigh_opts = {})

    main_target = sigh_opts
    if not main_target[:app_identifier]
      main_target = main_target.merge(:app_identifier => "com.company.product")
    end
    sticker_extension = main_target.merge(:app_identifier => main_target[:app_identifier] + ".Stickers")
    sigh(main_target)
    sigh(sticker_extension)
  end

  error do |lane, exception|
  end
end


``
</details>

<details><summary>`./fastlane/Appfile`</summary>

``ruby
app_identifier "com.company.product" # The bundle identifier of your app

team_name "Company Inc."
team_id "TEAMID"


``
</details>

### fastlane gems

| Gem      | Version | Update-Status      |
| -------- | ------- | ------------------ |
| fastlane | 2.60.1  | 🚫 Update available |


### Loaded fastlane plugins:

**No plugins Loaded**


<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| bundler                   | 1.12.5       |
| CFPropertyList            | 2.3.5        |
| i18n                      | 0.8.1        |
| minitest                  | 5.10.2       |
| thread_safe               | 0.3.6        |
| tzinfo                    | 1.2.3        |
| activesupport             | 4.2.8        |
| public_suffix             | 2.0.5        |
| addressable               | 2.5.2        |
| babosa                    | 1.0.2        |
| claide                    | 1.0.2        |
| fuzzy_match               | 2.0.4        |
| nap                       | 1.1.0        |
| cocoapods-core            | 1.2.1        |
| cocoapods-deintegrate     | 1.0.1        |
| cocoapods-downloader      | 1.1.3        |
| cocoapods-plugins         | 1.0.0        |
| cocoapods-search          | 1.0.0        |
| cocoapods-stats           | 1.0.0        |
| netrc                     | 0.7.8        |
| cocoapods-trunk           | 1.2.0        |
| cocoapods-try             | 1.1.0        |
| colored2                  | 3.1.2        |
| escape                    | 0.0.4        |
| fourflusher               | 2.0.1        |
| gh_inspector              | 1.0.3        |
| molinillo                 | 0.5.7        |
| ruby-macho                | 1.1.0        |
| nanaimo                   | 0.2.3        |
| xcodeproj                 | 1.5.2        |
| cocoapods                 | 1.2.1        |
| colored                   | 1.2          |
| highline                  | 1.7.8        |
| commander-fastlane        | 4.4.5        |
| declarative               | 0.0.10       |
| declarative-option        | 0.1.0        |
| unf_ext                   | 0.0.7.4      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20170404 |
| dotenv                    | 2.2.1        |
| excon                     | 0.59.0       |
| multipart-post            | 2.0.0        |
| faraday                   | 0.13.1       |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| faraday_middleware        | 0.12.2       |
| fastimage                 | 2.1.0        |
| jwt                       | 1.5.6        |
| little-plugger            | 1.1.4        |
| multi_json                | 1.12.2       |
| logging                   | 2.2.2        |
| memoist                   | 0.16.0       |
| os                        | 0.9.6        |
| signet                    | 0.7.3        |
| googleauth                | 0.5.3        |
| httpclient                | 2.8.3        |
| mime-types-data           | 3.2016.0521  |
| mime-types                | 3.1          |
| uber                      | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 3.1.1        |
| google-api-client         | 0.13.6       |
| json                      | 2.1.0        |
| mini_magick               | 4.5.1        |
| multi_xml                 | 0.6.0        |
| plist                     | 3.3.0        |
| rubyzip                   | 1.2.1        |
| security                  | 0.1.3        |
| slack-notifier            | 1.5.1        |
| terminal-notifier         | 1.8.0        |
| unicode-display_width     | 1.3.0        |
| terminal-table            | 1.8.0        |
| tty-screen                | 0.5.0        |
| word_wrap                 | 1.0.0        |
| rouge                     | 2.0.7        |
| xcpretty                  | 0.2.8        |
| xcpretty-travis-formatter | 0.0.4        |
</details>


*generated on:* **2017-12-28**
</details>

#######################################################################
# fastlane 2.71.0 is available. You are on 2.60.1.
# You should use the latest version.
# Please update using `bundle update fastlane`.
#######################################################################

2.71.0 Holiday release
* Add option to force device frame orientation (fixes #10928) (#11161) via Manu Wallner
* Testflight: New method to create group (#11321) via pav1790
* appledoc: accept multiple input paths (#11290) via Steven Hocking
* Fetch the correct prices from the new API in spaceship (#11314) via Ahmad A.Elghany
* Make swift code generation more aware of sensitive default values (#11301) via Joshua Liebowitz
* Add return type to get_version_number so it's accessible in Swift (#11315) via Joshua Liebowitz
* Enable rubocop cache on Circle (#11334) via Manu Wallner
* Flatten application_id in match (#11333) via Joshua Liebowitz
* Add a link to GitHub Working with Forks docs. (#11328) via Jimmy Dee
* Circle CI: Run in Ubuntu and macOS environments, in parallel (#11242) via Jan Piotrowski
* ErrorHandler isn't needed when building swift FastlaneRunner (#11326) via Joshua Liebowitz
* Android Appfile template: updated link to `supply` setup instructions (#11313) via Rishabh Tayal

2.70.3 Hotfix for Fastlane.Swift failing to run all commands
* Version bump to 2.70.3 (#11312) via Joshua Liebowitz
* Update support for `latest` for `download_dsyms` action (#10828) via Shai Mishali
* Verify all env variables are set for deployment (#11251) via Felix Krause
* Update yard development dependency (#11308) via Felix Krause
* Fix version selection with import_from_git (#11270) via Jimmy Dee
* If using system ruby, and json isn't installed, do the right thing (#11306) via Joshua Liebowitz
* Respect skip_type_validation when testing sample code. (#11305) via Mark Pirri
* [supply] replace hardcoded string `en-US` with actual language (#11296) via Jan Piotrowski

2.70.2 Bug fixes
* [supply] Improve check_superseded_tracks(#10927) via Marcelo Oliveira
* [supply] Add filename of Feature Graphic in output (#11283) via Luke Williams
* [build_ios_app] Use correct sdk name in build_ios_app docs (#11282) via Rob Moorman
* [snapshot] Expose the swift snapshot class to Objective-C (#11291) via Cédric Luthi
* [slack] Option to fail on errors posting slack notification (#11281) via Dan Cohn
* [produce] add missing app services (#11159) via Marius Rackwitz
* [Fastlane.swift] Fix missing return type for more actions (#11293) via Thi
* [Fastlane.swift] Fix gitBranch() has no return type (#11277) via Thi
* [Fastlane.swift] When generating Swift files, add new line to end of file (#11261) via Joshua Liebowitz
* [get_build_number] Fix missing return type (#11269) via Joshua Liebowitz
* [danger] Add option to run on a specific pull request (#11263) via Dan Cohn
* [fastlane core] Non-macOS platform: always return empty string as xcode_path (#11265) via Jan Piotrowski
* [release] Deploying fastlane should bump and commit fastlane swift files (#11289) via Joshua Liebowitz
* [release] Choose find syntax depending on OS (no find -E on Ubuntu) (#11264) via Jan Piotrowski
* [metrics] Fix app_id detection for swift projects (#11288) via Joshua Liebowitz
* [metrics] Default to unknown when app id is not available (#11285) via David Ohayon

To see all new releases, open https://github.com/fastlane/fastlane/releases

Please update using `bundle update fastlane`
bundler: failed to load command: fastlane (/Users/jenkins/.rbenv/versions/2.2.2/bin/fastlane)
FastlaneCore::Interface::FastlaneBuildFailure: [!] Error building the application - see the log above
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/fastlane_core/lib/fastlane_core/ui/interface.rb:153:in `build_failure!'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/fastlane_core/lib/fastlane_core/ui/ui.rb:14:in `method_missing'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/gym/lib/gym/error_handler.rb:70:in `handle_build_error'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/gym/lib/gym/runner.rb:90:in `block in build_app'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/fastlane_core/lib/fastlane_core/command_executor.rb:88:in `call'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/fastlane_core/lib/fastlane_core/command_executor.rb:88:in `execute'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/gym/lib/gym/runner.rb:86:in `build_app'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/gym/lib/gym/runner.rb:11:in `run'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/gym/lib/gym/manager.rb:14:in `work'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/fastlane/lib/fastlane/actions/gym.rb:65:in `run'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/fastlane/lib/fastlane/runner.rb:252:in `block (2 levels) in execute_action'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/fastlane/lib/fastlane/actions/actions_helper.rb:50:in `execute_action'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/fastlane/lib/fastlane/runner.rb:230:in `block in execute_action'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/fastlane/lib/fastlane/runner.rb:226:in `chdir'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/fastlane/lib/fastlane/runner.rb:226:in `execute_action'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/fastlane/lib/fastlane/runner.rb:148:in `trigger_action_by_name'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/fastlane/lib/fastlane/fast_file.rb:146:in `method_missing'
  Fastfile:45:in `block (2 levels) in parsing_binding'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/fastlane/lib/fastlane/lane.rb:33:in `call'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/fastlane/lib/fastlane/lane.rb:33:in `call'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/fastlane/lib/fastlane/runner.rb:201:in `try_switch_to_lane'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/fastlane/lib/fastlane/runner.rb:156:in `trigger_action_by_name'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/fastlane/lib/fastlane/fast_file.rb:146:in `method_missing'
  Fastfile:74:in `block (2 levels) in parsing_binding'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/fastlane/lib/fastlane/lane.rb:33:in `call'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/fastlane/lib/fastlane/lane.rb:33:in `call'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/fastlane/lib/fastlane/runner.rb:49:in `block in execute'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/fastlane/lib/fastlane/runner.rb:45:in `chdir'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/fastlane/lib/fastlane/runner.rb:45:in `execute'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/fastlane/lib/fastlane/lane_manager.rb:52:in `cruise_lane'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/fastlane/lib/fastlane/command_line_handler.rb:30:in `handle'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/fastlane/lib/fastlane/commands_generator.rb:104:in `block (2 levels) in run'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/commander-fastlane-4.4.5/lib/commander/command.rb:178:in `call'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/commander-fastlane-4.4.5/lib/commander/command.rb:178:in `call'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/commander-fastlane-4.4.5/lib/commander/command.rb:153:in `run'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/commander-fastlane-4.4.5/lib/commander/runner.rb:476:in `run_active_command'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:64:in `run!'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/commander-fastlane-4.4.5/lib/commander/delegates.rb:15:in `run!'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/fastlane/lib/fastlane/commands_generator.rb:303:in `run'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/fastlane/lib/fastlane/commands_generator.rb:42:in `start'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/fastlane/lib/fastlane/cli_tools_distributor.rb:66:in `take_off'
  /Users/jenkins/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/fastlane-2.60.1/bin/fastlane:20:in `<top (required)>'
  /Users/jenkins/.rbenv/versions/2.2.2/bin/fastlane:23:in `load'
  /Users/jenkins/.rbenv/versions/2.2.2/bin/fastlane:23:in `<top (required)>'
Build step 'Execute shell' marked build as failure
Archiving artifacts
```

**Comment by romk1n** [comment]
@blindmonkey have you ever resolved this issue ?

**Comment by blindmonkey** [comment]
@romk1n Yes, I was able to resolve the issue. I don't recall exactly how anymore; it's been a while since I've looked into this. It might have had something to do with not having defined either the profile or certificate in Apple's Developer dashboard (I may have been under the impression that they would be automatically created).

**Comment by romk1n** [comment]
@blindmonkey thanks!
